Source/WebKit/UIProcess/API/glib/WebKitInitialize.cpp

```diff
@@ -53,7 +53,7 @@ static void initializeRemoteInspectorServer()
             return nullptr;
 
         GUniquePtr<char> inspectorAddress(g_strdup(address));
-        char* portPtr = g_strrstr(inspectorAddress.get(), ":");
+        char* portPtr = g_strrchr(inspectorAddress.get(), ':');
```

Comment:
There is no `g_strrchr()` (which is why this doesn't build). You can change it to use `strrchr()` though.

Reply:
That said, the title of your commit message says "JavaScriptCore" and you are outside of it here, so this change seems misplaced regardless.